tests/platform_test.js

```diff
@@ -0,0 +1,175 @@
+  async ({ I, links }) => {
+    // Verify elements in login form
+    I.seeTextEquals('Login', 'legend');
+    I.seeTextEquals('Email', elements.emailFieldLabel);
+    I.seeInField(fields.emailField, '');
+    I.seeTextEquals('Password', elements.passwordFieldLabel);
+    I.seeInField(fields.passwordField, '');
+    I.seeAttributesOnElements(buttons.forgotPasswordLink, { href: links.forgotPassword });
+    I.seeAttributesOnElements(buttons.signIn, { disabled: true });
+    I.seeAttributesOnElements(buttons.goToSignUp, { disabled: null });
+    I.seeTextEquals('Sign up', buttons.goToSignUp);
+
+    // Focus on Email and Password fields to verify that fields are required
+    I.usePlaywrightTo('focus on email and password fields', async ({ page }) => {
+      page.focus('[data-qa="email-text-input"]');
+      page.focus('[data-qa="password-password-input"]');
+      page.focus('[data-qa="sign-in-to-sign-up-button"]');
+    });
+
+    pmmSettingsPage.perconaPlatform.verifyEmailFieldValidation();
+
+    // Password validation
+    I.seeTextEquals(messages.requiredField, elements.passwordValidation);
+
+    // Verify there is no validation for "pass" value
+    I.appendField(fields.passwordField, 'pass');
+    I.seeTextEquals('', elements.passwordValidation);
+  },
+);
+
+Scenario(
+  'PMM-T415 PMM-T842 Verify Sign Up form and validation @platform @settings',
+  async ({ I, links }) => {
+    // Open Sign Up Form
+    I.click(buttons.goToSignUp);
+    I.waitForVisible(elements.signUpForm, 30);
+
+    // Verify elements in Sign Up form
+    I.seeTextEquals('Sign up', 'legend');
+    I.seeTextEquals('Email', elements.emailFieldLabel);
+    I.seeInField(fields.emailField, '');
+    I.seeTextEquals('First name', elements.firstNameFieldLabel);
+    I.seeInField(fields.firstNameField, '');
+    I.seeTextEquals('Last name', elements.lastNameFieldLabel);
+    I.seeInField(fields.lastNameField, '');
+
+    I.seeTextEquals(messages.termsText, elements.termsCheckboxLabel);
+
+    // Verify Terms of Service and Privacy Policy links
+    await within(elements.termsCheckboxLabel, () => {
+      I.seeAttributesOnElements(locate('a').first(), { href: links.termsOfService });
+      I.seeTextEquals('Terms of Service', locate('a').first());
+      I.seeAttributesOnElements(locate('a').last(), { href: links.privacyPolicy });
+      I.seeTextEquals('Privacy Policy', locate('a').last());
+    });
+
+    // Verify Sign Up button is disabled and Back to Login button is present and enabled
+    I.seeAttributesOnElements(buttons.signUp, { disabled: true });
+    I.seeTextEquals('Back to login', buttons.backToSignIn);
+    I.seeAttributesOnElements(buttons.backToSignIn, { disabled: null });
+
+    // Focus on Email, First Name, Last Name fields and terms checkbox to verify that fields are required
+    I.usePlaywrightTo('focus on email and password fields', async ({ page }) => {
+      page.focus('[data-qa="email-text-input"]');
```

Comment:
FYI: you may want to use a function to save a few keystrokes :)
https://github.com/percona-platform/core-ui/blob/8f206e9ac5a8693b8bec60f65f5f1bdf2d3fcf54/src/shared/utils/test.utils.tsx#L4

Reply:
Importing `core-ui` requires installing a lot of other dependencies. I'd prefer to add our own function here if you don't mind